packages/modelviewer.dev/examples/scenegraph/index.html

```diff
@@ -478,6 +478,88 @@ <h2 class="demo-title">Swap textures</h2>
       </div>
     </div>
 
+
+    <div class="sample">
+      <div id="transformTexturesExample" class="demo"></div>
+      <div class="content">
+        <div class="wrapper">
+          <div class="heading">
+            <h2 class="demo-title">Transform textures</h2>
+            <p>As above, you can change these values in AR, but only in WebXR
+              mode. iOS Quick Look reflects these texture changes so long as the
+              USDZ is auto-generated.</p>
+          </div>
+          <example-snippet stamp-to="transformTexturesExample" highlight-as="html">
+            <template>
+<model-viewer id="helmet2" camera-controls touch-action="pan-y" src="../../shared-assets/models/glTF-Sample-Models/2.0/DamagedHelmet/glTF-Binary/DamagedHelmet.glb" ar alt="A 3D model of a helmet">
+  <div class="controls">
+      <p>Rotation: <span id="texture-rotation"></span></p>
+      <input type="range" min="0" max="0.2" value="0" step="0.01" id="rotationSlider">
+      <p>Scale: <span id="texture-scale"></span></p>
+      <input type="range" min="0.9" max="1.1" value="1" step="0.01" id="scaleSlider">
+      <p>Offset</p>
+      <input type="range" min="0.9" max="1.1" value="1" step="0.01" id="offsetSlider">
+    </div>
+</model-viewer>
+<script type="module">
+const modelViewerTexture2 = document.querySelector("model-viewer#helmet2");
+const rotationSlider = document.querySelector('#rotationSlider');
+const scaleSlider = document.querySelector('#scaleSlider');
+const offsetSlider = document.querySelector('#offsetSlider');
+
+modelViewerTexture2.addEventListener("load", () => {
+
+  const material = modelViewerTexture2.model.materials[0];
+
+  let rotationDisplay = document.querySelector('#texture-rotation');
+  let scaleDisplay = document.querySelector('#texture-scale');
+
+  rotationDisplay.textContent = rotationSlider.value;
+  scaleDisplay.textContent = scaleSlider.value;
+
+  const setTextureRotation = async (channel, rotation) => {
+    material[channel].setRotation(rotation);
+    rotationDisplay.textContent = rotation;
+  }
+
+  const setTextureScale = async (channel, scale) => {
+    material[channel].setScale(scale);
```

Comment:
We should probably explicitly set `wrap` to `REPEAT` to remind people that's important (it's already the glTF default, though apparently not on the helmet).